app/views/articles/show.html.slim

```diff
@@ -1,3 +1,6 @@
 h1 #{article.title}
 p
   | #{article.content}
+
+- if current_user&.administrative_role?
```

Comment:
Do we need to show this button only if article is unpublished?

Reply:
Yes, but I'm not sure how I will implement the "back to draft" button 😅

Reply:
@RasimKhusaenov What is the difficulty?  I can help if needed.  You need to implement action destroy in the Publications controller and a button with action destroy and text "Back to draft".

Reply:
These will be different buttons on the same page that will be displayed depending on the status of the article, right?

Reply:
@RasimKhusaenov Yes that's right.